src/vmm/src/persist.rs

```diff
@@ -376,38 +364,32 @@ pub enum ValidateCpuVendorError {
 /// - Failed to read host vendor.
 /// - Failed to read snapshot vendor.
 #[cfg(target_arch = "x86_64")]
-pub fn validate_cpu_vendor(microvm_state: &MicrovmState) -> Result<bool, ValidateCpuVendorError> {
-    let host_vendor_id = get_vendor_id_from_host()?;
-
-    let snapshot_vendor_id = microvm_state.vcpu_states[0]
-        .cpuid
-        .vendor_id()
-        .ok_or(ValidateCpuVendorError::Snapshot)?;
```

Comment:
I'd say this deserves an update in the changelog, because it is a user-visible change.

Reply:
Done